pyconpune/templates/schedule.html

```diff
@@ -0,0 +1,83 @@
+{% extends 'site_base.html' %}
+{% load staticfiles %}
+
+{% block body %}
+    <!-- About PyCon section -->
+    <section class="no-margin section-padding float-left full-width" id="full_schedule">
+        <div class="full-width">
+            <div class="col-960 col-center full-height">
+                <h2>Full Schedule</h2>
+
+                <ul class="nav nav-tabs nav-justified">
+                    {% for i in data %}
```

Comment:
Don't indent with django templating language (i.e., for, if, etc.) because the final rendered html then has a bad indentation.